clientv3/client.go

```diff
@@ -99,6 +101,17 @@ func (c *Client) Ctx() context.Context { return c.ctx }
 // Endpoints lists the registered endpoints for the client.
 func (c *Client) Endpoints() []string { return c.cfg.Endpoints }
 
+func (c *Client) UpdateEndpoints(eps ...string) {
+	c.cfg.Endpoints = eps
```

Comment:
this is a code duplication with line164-175. @heyitsanthony might have some idea about this.